docs/2/technical/data-format/overview.md

```diff
@@ -210,17 +197,12 @@ The origin can be interesting when the data is shared with you to understand whe
 },
 ```
 
-The ownership of the data is an essential part of the data, here you will find who owns the data, which account (`user`) and under which identity (`org`). So in case of doubt for the creation of a share rule these data are essential to share from the right place the right data. 
+The ownership of the data is an essential part of the data, here you will find who owns the data, which account (`user`) and under which identity (`org`). In case there is doubt surrounding the creation of a share rule, this section helps to clarify that the right data is being shared correctly. 
```

Comment:
There may be multiple owners for every piece of data. Owners have the ability to create share Rules that grant privileges to other users.